nuxeo-features/nuxeo-elasticsearch/nuxeo-elasticsearch-audit/src/test/java/org/nuxeo/elasticsearch/TestESHistoryProvider.java

```diff
@@ -429,6 +435,29 @@ public void testCustomDocumentHistoryPageProvider() {
         assertEquals(5, entries.size());
     }
 
+    @Test
+    public void testCustomFixedPartDocumentHistoryPageProvider() {
+        assertNotNull(pageProviderService.getPageProviderDefinition("FIXED_PART_DOCUMENT_HISTORY_PROVIDER"));
+        DocumentModel searchDoc = session.createDocumentModel("BasicAuditSearch");
+        searchDoc.setPathInfo("/", "auditsearch");
+        searchDoc = session.createDocument(searchDoc);
+
+        searchDoc.setPropertyValue("basicauditsearch:eventIds", null);
+        searchDoc.setPropertyValue("basicauditsearch:eventCategories", null);
+        searchDoc.setPropertyValue("basicauditsearch:startDate", null);
+        searchDoc.setPropertyValue("basicauditsearch:endDate", null);
+
+        //test with doc
+        PageProvider<LogEntry> pageProvider = getPageProvider("FIXED_PART_DOCUMENT_HISTORY_PROVIDER", 30, 0, doc);
+        pageProvider.setSearchDocumentModel(searchDoc);
+        assertEquals(1, pageProvider.getCurrentPage().size());
+
+        //test with proxy to check that the doc uuid is correctly set in the fixed part
```

Comment:
This comment and the one above has the correct space after `//` on master.

Reply:
Thanks for catching it.